CONTRIBUTING.md

```diff
@@ -99,3 +99,13 @@ $ npm run dev
 ```
 
 Once setup is complete, you can refer to VuePress documentation on the actual implementation here: https://vuepress.vuejs.org/guide/.
+
+### Git/GitHub workflow
+
```

Comment:
Could we have a one-line intro to the section here? Just something to the effect of "this is our preferred process for how to make a PR on github"

Reply:
Absolutely, done.